### PR TITLE
Switch from kunihiko-t/review-request-action to necojackarc/auto-request-review

### PR DESCRIPTION
**Pull request by ktsamis**
## What does this PR change?

Switch from kunihiko-t/review-request-action to necojackarc/auto-request-review as MR

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
- No tests: already covered

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by ktsamis** [comment]
@jordimassaguerpla Also if we can use this https://github.community/t/resource-not-accessible-by-integration-assigning-author/136391 we should be able to enable this for PRs from forks. Are there any security implications from this? I don't see anything wrong with running the action with GITHUB_TOKEN being writeable while the action is running.

File: .github/reviewers.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - maximenoel8
+      - lkotek
+      - srbarrios
+      - Bischoff
+      - calancha
+
+files:
+  # Keys are glob expressions.
+  # You can assign groups defined above as well as GitHub usernames.
+  'testsuite/*':
```
[the remark this answers is not in the document]
Comment (ktsamis) [reply .github/reviewers.yml]: Fixed, thanks, very important catch

**Review by calancha — APPROVED** [review]

**Review by srbarrios — APPROVED** [review]
This will for sure help with maintainability.
Now let's see if the github action will work with forks, maybe using pull_request_target.

File: .github/workflows/auto_request_review.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: Auto Request Review
+
+on:
+  pull_request:
```
Comment (srbarrios) [inline .github/workflows/auto_request_review.yml:4]:
```suggestion
  pull_request_target:
```
As far as I read in your shared post, that change might make the trick to work with forks.
It's worth trying. Could you remove us as reviewers, make this change, and see how it behaves?

**Review by calancha — CHANGES_REQUESTED** [review]
Changed to 'request changes; to let the author fix other issues.

**Pull request merged**